vscode-trace-extension/src/utils/trace-server-status.ts

```diff
@@ -16,20 +19,34 @@ export class TraceServerConnectionStatusService {
     };
 
     public async updateServerStatus(status: boolean): Promise<void> {
+        if (status === this._status) {
+            return;
+        }
+        this._status = status;
+
         await vscode.commands.executeCommand('setContext', 'traceViewer.serverUp', status);
+
+        const command = VSCODE_MESSAGES.CONNECTION_STATUS;
+        const data = this._status.toString();
+
+        traceExtensionWebviewManager.getAllActiveWebviews().forEach(_view => {
+            _view.webview.postMessage({ command, data });
```

Comment:
I'm wondering if this is still required, after the recent commit where webviews are disposed automatically (https://github.com/eclipse-cdt-cloud/vscode-trace-extension/commit/77bf4f03c44677685bcb4ae92003d7f43f3a0be8)?

Debugging through the code, it seems that on trace server shutdown, the webviews are already disposed by the time this is executed, and on trace server startup, the webviews are freshly created and may not need this update?

Reply:
I agree.  I added this to be thorough and send to all webviews, but
a) the explorer webviews should be disposed when the welcome screen is rendered (what you said)
b) the webviews don't have any listeners for this event.

I went ahead and removed this for now.  It can be re-added in the future if necessary.